eng/scripts/Language-Settings.ps1

```diff
@@ -217,41 +217,6 @@ function Get-DocsMsPackageName($packageName, $packageVersion) {
   return "$(Get-PackageNameFromDocsMsConfig $packageName)@$packageVersion"
 }
 
-
-# Performs package validation for a list of packages provided in the doc 
-# onboarding format ("name" is the only required field): 
-# @{
-#   name = "@azure/attestation@dev";
-#   folder = "./types";
-#   registry = "<url>";
-#   ...
-# }
-function ValidatePackagesForDocs($packages, $DocValidationImageId) {
-  # Using GetTempPath because it works on linux and windows
-  $tempDirectory = Join-Path ([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName())
-  New-Item -ItemType Directory -Force -Path $tempDirectory | Out-Null
-
-  $scriptRoot = $PSScriptRoot
-  # Run this in parallel as each step takes a long time to run
-  $validationOutput = $packages | ForEach-Object { [PSCustomObject]$_ } | Foreach-Object -Parallel {
-    # Get value for variables outside of the Foreach-Object scope
-    $scriptRoot = "$using:scriptRoot"
-    $workingDirectory = "$using:tempDirectory"
-    Write-Host "`"$scriptRoot\validate-docs-package.ps1`" -Package $_ -DocValidationImageId `"$($using:DocValidationImageId)`" -WorkingDirectory $workingDirectory"
-    return ."$scriptRoot\validate-docs-package.ps1" -Package $_ -DocValidationImageId "$using:DocValidationImageId" -WorkingDirectory $workingDirectory 
-  }
-
-  # Clean up temp folder
-  Remove-Item -Path $tempDirectory -Force -Recurse -ErrorAction Ignore | Out-Null
-
-  return $validationOutput
-}
-
-$PackageExclusions = @{
-  '@azure/identity-vscode'            = 'Fails type2docfx execution https://github.com/Azure/azure-sdk-for-js/issues/16303';
-  '@azure/identity-cache-persistence' = 'Fails typedoc2fx execution https://github.com/Azure/azure-sdk-for-js/issues/16310';
-}
-
 function Update-javascript-DocsMsPackages($DocsRepoLocation, $DocsMetadata, $DocValidationImageId) {
```

Comment:
Do you plan to clean out this use of the docker image?

Reply:
The docker image is still referenced in files in eng/common. The `Update-javascript-DocsMsPackages` function can be removed (but I'm not doing it in this PR)